src/Simulation/QsharpFoundation/Utils.qs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Intrinsic {
```

Comment:
It doesn't have any user-facing impact, but I'm not sure the name "Utils.qs" communicates to me that this source file contains definitions for two intrinsic callables that are classical, by contrast with the quantum intrinsics in Intrinsic.qs.

Reply:
I struggled to come up with a name for this, actually... the only clearer option I could think of would be separating them into Message.qs and Random.qs and having only one operation per file, which feels silly to do for body-intrinsic methods where there isn't any complicated implementation. Anyone have any suggestions here?

Reply:
You know, the more I think about it the more I like splitting them anyway, even if it does make very small files. This will kind of line up with what I'm imagining later parts of this decomposition work will look like anyway.